start/data/data-transfer-with-globus.md

```diff
@@ -28,28 +27,24 @@ Transfer data between existing servers
     the View Transfer window to see your transfer progress.  Click on the
     progress bar to drill in on transfer details.
 
-Transferring between your laptop and OSG Connect Stash
-------------------------------------------------------
+## Transferring between your laptop and OSG Connect Stash
 
 To add your own laptop as an endpoint, so that you can transfer data to/from
 stash, you need to install Globus Connect, by clicking on the "Get Globus
 Connect" in the upper right corner of the Start Transfer page. More detailed
```

Comment:
repeat instructions for accessing the "Start Transfer" page, or reference the instructions above?